modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -20,39 +20,36 @@ However, to do so, you have to set `'sharing.federation.allowHttpFallback' => tr
 
 Follow these steps to establish a trusted connection between two servers.
 
-. Verify that both servers have SSL certificates.
-  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid.
+. Verify that both servers have SSL certificates
+  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid
 +
 .Lock icon in the address bars in Firefox, Google Chrome, and Safari.
 image:configuration/files/browser-address-bars.png[Lock icon in Firefox, Google Chrome, and Safari]
-. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>' setting is configured to the correct URL, instead of `localhost`, in **config.php**.
-. Reset the federation job in your `oc_jobs` table. 
-  This job is required to get the verification token from the other server to establish a federation connection between two servers. 
-  The resetting ensures that it will be executed when we run xref:configuration/server/background_jobs_configuration.adoc#cron[system:cron] later.
+. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>'` setting is configured to the correct URL instead of `localhost` in **config.php**
+. Navigate to **admin settings -> Sharing -> Federation**
+. Add **server 1** to the trusted servers on **server 2**
+. Add **server 2** to the trusted servers on **server 1**
+. Run `system:cron` from the ownCloud directory of both servers (for example `/var/www/owncloud/`)
 +
-[source,bash]
+[source,bash,subs="attributes+"]
 ----
-mysql -u root -e "update oc_jobs set last_run=0 where class='OCA\\Federation\\SyncJob';" owncloud;
-mysql -u root -e "update oc_jobs set last_checked=0 where class='OCA\\Federation\\SyncJob';" owncloud;
+{occ-command-example-prefix} system:cron
 ----
-. Navigate to **admin settings -> sharing -> Federation**
-. Add **server 1** to the trusted servers on **server 2**.
-. Add **server 2** to the trusted servers on **server 1**.
-. Now run the cron job in your ownCloud directory (for example `/var/www/owncloud/`).
+. Run the `OCA\Federation\SyncJob` job with the `force` option on both servers (you can get the correspondent job ID by using the `background:queue:status` occ command)
 +
 [source,bash,subs="attributes+"]
 ----
-{occ-command-example-prefix} system:cron
+{occ-command-example-prefix} background:queue:execute --force 10
```

Comment:
Is that sufficient?
I am not sure, if maybe we still need to call occ system:cron 
t really execute the job?

Reply:
Based on my tests it seems to be sufficient.

Reply:
we cannot be sure that id 10 will be what we need. it varies per system

Reply:
correct but there is also documented `you can get the correspondent job ID by using the background:queue:status occ command`.

Reply:
How about something like:
```
fedJobId=$(occ background:queue:status | grep Federation.SyncJob | cut -d '|' -f 2)
occ background:queue:execute --force $fedJobId
```

Reply:
I've checked a few of my systems, it is always 10 there. I really thought that is a constant. :frowning:

Reply:
It never works for me on the first attempt.
I have now succeeded in one case, where I did exactly as described here, 
got only a yellow marker in the webUI, then pondered a while, and re-ran (a few minutes later) 
`occ background:queue:execute --force $fedJobId`
This additional (or delayed) command then did the magic: It added the entry to oc_addressbooks, and the trusted server became green in the UI.